spec/requests/games_spec.rb

```diff
@@ -107,15 +107,42 @@
     end
 
     context "when game is finished" do
+      let(:rico)  { Player.create!(nickname: "Rico") }
+      let(:zim)   { Player.create!(nickname: "Zim") }
+
       before :each do
         game.update(state: "finished")
-        Player.create!(nickname: "Zim", game: game)
-        Player.create!(nickname: "Rico", game: game, won: true)
 
+        rico.update!(game: game, won: true)
+        zim.update!(game: game, won: false)
+      end
+
+      it 'displays that game is finished' do
         get "/games/#{game.to_param}"
+        expect(response.body).to include("finished")
+      end
+
+      context 'wen signed in player won the game' do
```

Comment:
So I'd say `when the signed in player`.
Because any other signed in player won't see this message.